### ISSUE TITLE
[Feature]: I would like to show a caret programmatically.

### ISSUE THREAD

**Issue by AdvancedNotifier** [MicrosoftEdge/WebView2Feedback#4142]
### Describe the feature/enhancement you need

I would like to show the caret inside the WebView2 in my .Net app.

To try to do this, I used the CoreWebView2Environment with AdditionalBrowserArguments of "--enable-caret-browsing" and "--blink-settings=caretBrowsingEnabled", described [here](https://peter.sh/experiments/chromium-command-line-switches/#enable-caret-browsing). But this does not work.

It would be nice if you would implement this browser argument.

### The scenario/use case where you would use this feature

I'm trying to allow the users of my app to modify the shown HTML page directly.

### How important is this request to you?

Impactful. My app's user experience would be significantly compromised without it.

### Suggested implementation

See above.

### What does your app do? Is there a pending deadline for this request?

My app is an E-Mail client, which should allow the user to compose E-Mails in HTML.

**Comment by AdvancedNotifier** [comment]
In the meantime, I found the more compfortable `contententable` feature and I can use them.

But the `AreBrowserAccelatorKeysEnabled` was `true` while I tried to use the `--enable-caret-browsing` option.

**Comment by monica-ch** [comment]
@AdvancedNotifier Glad you found a solution. 

The mentioned feature flag is working for me from our sample app without any issue. Could you please try with our sample app and see if it still is working?

**Comment by boreno** [comment]
Hello, I have the exact same need. I am not able to enable caret browsing in the WebView2 control (runtime 120.0.2210.61, SDK build 2194.0) through the --enable-caret-browsing parameter. I have also tried this in the WebView2APISample project.

Note: If I *don't* specify --enable-caret-browsing, I am able to enable caret browsing by pressing F7. I have found no way to enable caret browsing programmatically without a confirmation dialog.
